contrib/debian/rules

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+
+# pie causes build failures
+export DEB_BUILD_MAINT_OPTIONS = hardening=+all,-pie
```

Comment:
Yeah I'd like to keep it  enabled, but when switching to meson/ninja this came up and that was the temporary solution.
I'll turn it off on the next upload to experimental to confirm if it's still a problem (test build on x86-64 seems to indicate that it shouldn't be a problem right now).